JASP-Desktop/po/jasp_nl.po

```diff
@@ -16078,10 +15845,2124 @@ msgctxt "RibbonModel|"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Estimated studies' effects"
+msgstr "Geschatte onderzoekseffecten"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Plots"
+msgstr "Grafieken"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Pooled estimates"
+msgstr "Samengebrachte schattingen"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Forest plot"
+msgstr "Bosgrafiek"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Show"
+msgstr "Toon"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Observed"
+msgstr "Waargenomen"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Estimated"
+msgstr "Geschat"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Both"
+msgstr "Beiden"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Ascending"
+msgstr "Toenemend"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Descending"
+msgstr "Afnemend"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Row order"
+msgstr "Rijvolgorde"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Weights"
+msgstr "Gewichten"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Type"
+msgstr "Type"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Model-averaged"
+msgstr "Modelgemiddeld"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Conditional"
+msgstr "Conditioneel"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Show priors"
+msgstr "Toon priors"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Conditional models only"
+msgstr "Alleen de geconditioneerde modellen"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Model number"
+msgstr "Model nummer"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "MCMC Diagnostics"
+msgstr "MCMC Diagnostiek"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Overview"
+msgstr "Overzicht"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Include theta"
+msgstr "Includeer theta"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Plot"
+msgstr "Grafiek"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Trace"
+msgstr "Spoor"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Autocorrelation"
+msgstr "Autocorrelatie"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Posterior samples densities"
+msgstr "Posterior steekproefdichtheid"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Transform estimates"
+msgstr "Transformeer schattingen"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Advanced"
+msgstr "Geavanceerd"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Transform correlations"
+msgstr "Getransformeerde correlatie"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Cohen's d"
+msgstr "Cohen's d"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Fisher's z"
+msgstr "Fisher's z"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Bridge sampling iterations:"
+msgstr "Bridge sampling iteraties:"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Estimation settings (MCMC)"
+msgstr "Schattingsinstellingen (MCMC)"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Adaptation"
+msgstr "Aanpassing"
+
+msgctxt "RobustBayesianMetaAnalysis|"
+msgid "Burnin"
```

Comment:
Should the original not be "Burning" but ah oh, we don't include this analysis anyway.